libopae/plugins/xfpga/opae_ioctl.c

```diff
@@ -49,6 +49,9 @@ fpga_result opae_ioctl(int fd, int request, ...)
 		case EINVAL:
```

Comment:
Should line 47 be OPAE_ERR?

Reply:
I'm going off previous implementation which used MSG instead of ERR.
Perhaps the caller should determine if the failure to this warrants an error message?